.circleci/documentation-versions.py

```diff
@@ -19,17 +45,27 @@ def main():
     except IndexError:
         sys.exit("Missing build dir path: python /path/docs")
 
-    ignored_dirs = {".git"}
-    dirs = [
-        item
-        for item in os.scandir(build_dir)
-        if item.is_dir() and item.name not in ignored_dirs
-    ]
-    sorted_dirs = sorted(dirs, reverse=True, key=attrgetter("name"))
-    active_versions = [item.name for item in sorted_dirs[:7]]
-    versions_to_remove = (item.path for item in sorted_dirs[7:])
+    documentation_directories = sorted(
+        (
+            item
+            for item in os.scandir(build_dir)
+            if item.is_dir() and VERSION_REGEX.match(item.name)
+        ),
+        reverse=True,
+        key=attrgetter("name")
+    )
+
+    active_versions = (
+            ["latest", "dev"]
+            + [item.name for item in documentation_directories[:NUMBER_OF_VERSIONS_TO_KEEP]]
+            + KEEP_VERSIONS
+    )
+    versions_to_remove = (
+        item.path
+        for item in documentation_directories[NUMBER_OF_VERSIONS_TO_KEEP:]
+        if item.name not in KEEP_VERSIONS
+    )
     with open(build_dir + "/version-helper.js", "w+") as outf:
-        # select 7 latest versions
         outf.write("let versions = {};".format(active_versions))
     # print out old versions to be removed
```

Comment:
maybe make the comment "print versions to remove to stout for deletion by bash script"